### PR TITLE
[Fixes #128] Emphasize CC license on photos

### LINKED ISSUE #128: Add a license notice for uploads and data
Probably CC BY

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

**Comment by comzeradd** [comment]
@akatsoulas quick r? :)

File: marketpulse/main/templates/media_new.html [hunk 1]
```diff
@@ -57,6 +57,11 @@ <h2 class="title">New Photo</h2>
             <div>
               <a href="#" class="btn btn-default btn-block" id="pickimage" data-role="button">Pick Image</a>
             </div>
+            <div class="license-text">
+              all photo contributions are available under a
```
Comment (johngian) [inline marketpulse/main/templates/media_new.html:61]: I suggest the text to be:
`All photo contributions are available under CC-BY-SA`

**Comment by johngian** [comment]
r+wc

File: marketpulse/main/templates/media_new.html (hunk 1 above)
Comment (comzeradd) [inline marketpulse/main/templates/media_new.html:61]: I prefer to have more or less the same text we have on the footer. Not all people are aware of CC initials.

**Pull request merged**